CMakeLists.txt

```diff
@@ -150,6 +150,10 @@ if(OGS_BUILD_TESTS)
     set(Data_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/Tests/Data CACHE INTERNAL "")
 endif()
 
+if(OGS_BUILD_UTILS)
+    include(${CMAKE_SOURCE_DIR}/Applications/Utils/ModelPreparation/PartitionMesh/MetisSetup.cmake)
```

Comment:
I would move the `include` to `Applications/Utils/ModelPreparation/PartitionMesh/CMakeLists.txt`